modules/publicize/publicize-jetpack.php

```diff
@@ -1,13 +1,16 @@
 <?php
 
+jetpack_require_lib( 'class.jetpack-service-helpers' );
```

Comment:
You forgot to rename this to `class.jetpack-service-helper`

Reply:
Also this should be moved to the constructor of Publicize, otherwise you're loading the library on every request, even if it isn't needed.